README.md

```diff
@@ -1,10 +1,17 @@
 ## C# for Visual Studio Code (powered by OmniSharp)
 
-Don't install this extension unless you want to try an early version. By default it is already
-bundled with Visual Studio Code.
+Don't install this extension unless you want to try an early version. 
 
-### Development
+This is a preview of C# support in Visual Studio code. It is designed to work well with upcoming [.NET Core 1.0]([link](http://dotnet.github.io/). 
```

Comment:
I think you want to remove the '[link]' text after '[.NET Core 1.0]'. I am not sure why it works.